ui/main/src/app/modules/card/components/card-ack/card-ack.component.ts

```diff
@@ -102,6 +104,38 @@ export class CardAckComponent implements OnInit, OnChanges, OnDestroy {
 
     }
 
+    private integrateChildCardsInRealTime() {
+        OpfabStore.getLightCardStore()
+            .getNewLightChildCards()
+            .pipe(
+                takeUntil(this.unsubscribe$),
+                map((lastCardLoaded) => {
+                    this.updateAcknowledgeButtonVisibilityIfNeeded(lastCardLoaded);
+                })
+            )
+            .subscribe();
+
+        OpfabStore.getLightCardStore()
+            .getDeletedChildCardsIds()
+            .pipe(
+                takeUntil(this.unsubscribe$),
+                map((lastCardDeleted) => {
+                    this.updateAcknowledgeButtonVisibilityIfNeeded(lastCardDeleted);
+                })
+            )
+            .subscribe();
+    }
+
+    private updateAcknowledgeButtonVisibilityIfNeeded(childCard) {
```

Comment:
i would suggest naming it " updateAcknowledgeButtonVisibilityIfCardsIsChildOfCurrentCard"